### ISSUE TITLE
Ports build seems broken

### ISSUE THREAD

**Issue by darkoverlordofdata** [tagattie/FreeBSD-VSCode#34]
I had to reinstall freebsd. VSCode is no longer in the repo, and it seems the port build is broken...
After running for about 24 hrs, I get this error:

```
===>  Building for vscode-1.56.2_1
cd /usr/ports/editors/vscode/work/vscode-1.56.2 && /usr/bin/env BUILD_SOURCEVERSION=054a9295330880ed74ceaedda236253b4f39a335  XDG_CACHE_HOME=/usr/ports/editors/vscode/work/.cache ELECTRON_SKIP_BINARY_DOWNLOAD=1 PLAYWRIGHT_BROWSERS_PATH=/usr/ports/editors/vscode/work/pw-browsers  PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 PYTHONDONTWRITEBYTECODE=1 XDG_DATA_HOME=/usr/ports/editors/vscode/work  XDG_CONFIG_HOME=/usr/ports/editors/vscode/work  HOME=/usr/ports/editors/vscode/work TMPDIR="/usr/ports/editors/vscode/work" PATH=/usr/ports/editors/vscode/work/.bin:/sbin:/bin:/usr/sbin:/usr/bin:/usr/games:/usr/local/sbin:/usr/local/bin NO_PIE=yes MK_DEBUG_FILES=no MK_KERNEL_SYMBOLS=no SHELL=/bin/sh NO_LINT=YES LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 PREFIX=/usr/local  LOCALBASE=/usr/local  CC="cc" CFLAGS="-O2 -pipe  -fstack-protector-strong -fno-strict-aliasing "  CPP="cpp" CPPFLAGS=""  LDFLAGS=" -fstack-protector-strong " LIBS=""  CXX="c++" CXXFLAGS="-O2 -pipe -fstack-protector-strong -fno-strict-aliasing  "  MANPREFIX="/usr/local" BSD_INSTALL_PROGRAM="install  -s -m 555"  BSD_INSTALL_LIB="install  -s -m 0644"  BSD_INSTALL_SCRIPT="install  -m 555"  BSD_INSTALL_DATA="install  -m 0644"  BSD_INSTALL_MAN="install  -m 444"  yarn --frozen-lockfile --offline
yarn install v1.22.10
$ node build/npm/preinstall.js
[1/4] Resolving packages...
success Already up-to-date.
$ node build/npm/postinstall.js
Installing dependencies in build...
$ yarn --frozen-lockfile --offline
yarn install v1.22.10
[1/4] Resolving packages...
[2/4] Fetching packages...
error Can't make a request in offline mode ("https://registry.yarnpkg.com/@azure/cosmos/-/cosmos-3.9.3.tgz")
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
*** Error code 1

Stop.
make: stopped in /usr/ports/editors/vscode
/usr/ports/editors/vscode $ 
```

**Comment by darkoverlordofdata** [comment]
I tried what i've done before,  download the old release and 
```
sudo pkg install ./vscode-1.41.0-freebsd12-amd64.txz  
```

This used to work.  Now I get:
```
code-oss
ld-elf.so.1: /usr/local/share/code-oss/bin/../code-oss: Undefined symbol "_ZN3re23RE23Arg12parse_stringEPKcmPv"
```

I'm running on 12.2 release:

FreeBSD sudanna 12.2-RELEASE FreeBSD 12.2-RELEASE r366954 GENERIC  amd64

**Comment by darkoverlordofdata** [comment]
I was having problems with chromium also. And I guess I have a workaround - I installed GhostBSD, and I am able to install VSCode there via the package manager. It's a prior version. I guess the breakage hasn't moved there yet.  It has to be the repo - I installed the same FreeBSD iso a month ago, and it worked and I could install these programs then. But not today.... I may have to rethink using FreeBSD as a desktop. I didn't realize it was so flakey.
